draft-ietf-rats-architecture.md

```diff
@@ -376,6 +376,76 @@ create Evidence. Examples of environments that can be used as
 Attesting Environments include Trusted Execution Environments (TEE),
 embedded Secure Elements (eSE), or Hardware Security Modules (HSM).
 
+## Layered Attestation Procedures
+
+By definition, the Attester role takes on the duty to create Attestation Evidence.
+The fact that an Attester role is composed of several types of environments that
+can be nested or staged adds complexity to the architectural layout of how an
+Attester -- in itself -- is composed and therefore has to conduct the Claims Collection
+in order to create believable Attestation Evidence. This complexity is
+independent of the concept defined by Composite Attesters.
```

Comment:
Maybe the point at the end, about how a composite attester can each produce their own *signed* evidence could be brought to the top.

Reply:
I am unsure, what the final decision on this is / will be. Will have to discuss